corehq/apps/events/views.py

```diff
@@ -526,3 +530,33 @@ def paginated_attendees(request, domain):
         'attendees': [{'case_id': c.case_id, 'name': c.name} for c in cases],
         'total': total,
     })
+
+
+@require_GET
+@login_and_domain_required
+@require_permission(HqPermissions.manage_attendance_tracking)
+def get_attendees_and_attendance_takers(request, domain):
+    location_id = request.GET.get('location_id', None)
+    if location_id:
+        attendees = AttendeeModel.objects.by_location_id(domain=domain, location_id=location_id)
+        attendance_takers = get_mobile_users_by_filters(
+            domain,
+            {'location_id': location_id}
+        )
+    else:
+        attendees = AttendeeModel.objects.by_domain(domain=domain)
+        attendance_takers = get_all_commcare_users_by_domain(domain)
```

Comment:
The docstring for `get_all_commcare_users_by_domain` says `Returns all CommCareUsers by domain regardless of their active status`. I see we use this in `_get_possible_attendance_takers_ids` as well. This might become an issue. I think we can use `get_mobile_users_by_filters` with the `user_active_status` filter set